src/main/resources/files/readme.md.tpl

```diff
@@ -112,7 +112,19 @@ The example calls one endpoint from another JAX-RS resource where generated Rest
 [# th:if="${mp_graphql}"]
 ### GraphQL
 
-GraphQL is a query language for APIs and a library for fulfilling those queries with your existing data. Specification [here](https://microprofile.io/project/eclipse/microprofile-graphql)
-
-The example contains a limited version of the SuperHero example from the TCK and shows the Schema file for it.
+GraphQL is a remote data query language initially invented by Facebook and now evolving under it's own specification and community. MicroProfile GraphQL provides annotation-based APIs for building GraphQL services in Java. The specification is available [here](https://microprofile.io/project/eclipse/microprofile-graphql).
+
+The example contains a limited version of the SuperHero example from the TCK and shows the Schema file for it. You can think of it as something a comic-book government agency might use to keep track of Super Heroes.
+
+To get started, run the sample, then browse to: http://localhost:9080/graphql-ui - you can then issue a query of all known super heroes like this:
```

Comment:
9080 is the port for Liberty. Other runtime uses different ports. Please replace 9080 with `[#th:text="${port_service_a}"/]`

Reply:
the GraphQL-ui is not required according to the spec (if I'm not mistaken).  So it might not be available even if the runtime has support for GraphQL.

Reply:
Thanks for the review comments - they both make sense.  I wonder if I could put that section in a `[# th:if="${mp_servername} == 'liberty'"]` block.  I could still add the `port_service_a` variable too just in case the user specifies a different port.

Reply:
@andymc12 See #356 where I populated the `mp_servername`.

Reply:
@rdebusscher I saw other places in the code base where they used a similar pattern.  Will my most recent commit work? (It seems to work for me locally, but I don't really have a way of testing MP GraphQL in a different, non-Liberty container)

Reply:
@andymc12 There seems no other implementations that contain the implementation have passed the TCK. https://wiki.eclipse.org/MicroProfile/Implementation#MP_GraphQL_implementations

Reply:
@rdebusscher thanks for pointing that out.  I believe Wildfly and Quarkus (via extension) both also pass the TCK - and I think Helidon is also close.  I'll let those folks know so that they can update the implementations page and can optionally update the starter too.  Thanks again!